src/pages/patientView/timeline/legacy.js

```diff
@@ -166,6 +166,7 @@ export function buildTimeline(params, caseIds, patientInfo, clinicalDataMap, cas
     window.pvTimeline =
         window.pvTimeline
             .splitByClinicalAttributes("Treatment", ["SUBTYPE", "AGENT"])
+            .splitByClinicalAttributes("Treatment", ["TREATMENT_TYPE", "AGENT"])
```

Comment:
Thanks for checking! It's splitting the treatment track by either subtype & agent or treatment_type & agent. Only if all events in that track have values for the given attributes. That's the way `splitByClinicalAttributes` works. I think it's best to try and split the treatment timelines as much as we can to make it more easy to interpret.